### DISCUSSION TITLE
Schedule not triggered as expected

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17790] [Q&A]
Howdy fellow dagsterites. I am encountering some issues with having jobs scheduled on an hourly basis. I think I've followed the process correctly.

I am defining the Schedule as this:
```
sentimax_compute_job = define_asset_job("sentimax_compute_job", selection=AssetSelection.groups("sentimax_compute"))

sentimax_compute_schedule = ScheduleDefinition(job=sentimax_compute_job, 
                                               cron_schedule=HOURLY_PLUS.format('0'),
                                               execution_timezone="Australia/Sydney")
```
And passing to the Definition as this:
```
defs = Definitions(
    assets=all_assets,
    jobs=[sentimax_compute_job],
    schedules=[sentimax_compute_schedule],
    resources={
        "dbt": DbtCliResource(project_dir=DBT_PROJECT_DIR),
        "bigquery": BigQueryResource(
            project=EnvVar("GCP_PROJECT"),
            dataset="SENTIMAX",
         ),
        "io_manager": BigQueryPandasIOManager(project=EnvVar("GCP_PROJECT"),
                                              dataset="SENTIMAX")
    },
)
```
Everything seems fine in the UI, but nothing gets triggered when the hour mark ticks over. I can perform manual runs fine, just not scheduled. Any ideas on how to troubleshoot this?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699392441467409?thread_ts=1699392441.467409&cid=C01U954MEER).*

**Comment by jamiedemaria** [comment]
Are you seeing any information on the schedules page?

A couple things to check:
1. confirm the schedule has been turned on 
2. Is there anything listed in the "Tick History"?

**Reply by GaryPate** [reply]
Schedule turned on

![image](https://github.com/dagster-io/dagster/assets/20076884/a4fbe8a1-5fc9-4103-88b3-d898c1dd9417)

Nothing in tick history and says schedule has never run
![image](https://github.com/dagster-io/dagster/assets/20076884/8c460bcb-f876-4271-b89f-a4184cf341b6)

**Reply by jamiedemaria** [reply]
ok - does anything unexpected happen when you click the "Test Schedule" button? Are there any logs from the daemon?

**Reply by GaryPate** [reply]
Update

I was able to get scheduled execution working after rolling back to version 1.5.4 and switching on the following in the UI:
- Experimental schedule/sensor logging view
- New asset lineage sidebar

If I get time, I will try to reproduce and file a bug.
